credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -201,3 +202,20 @@ def _remove_initial_junk(self, snippet):
             r"^((\s*|\ *)\@\@.*\@\@(\s*|\ *)|(\s*|\ *)\+(\s*|\ *)|(\s*|\ *)\-(\s*|\ *)|(\s*|\ *))",
             "",
             snippet).strip()
+
+    def _assignment_expression(self, snippet):
+        """ Check if the snippet is an assignment or not.
+        """
+        assignment_symbols = ['=', ':=', '<-', ':']
```

Comment:
is `<-` correct? isn't it on the other way?

`myclass -> myfield = "somevalue"`

Reply:
moreover, I wonder if we are missing some other way to assign values. E.g., `strcpy` calls in c